release-notes/9.0/supported-os.json

```diff
@@ -14,84 +15,44 @@
                         "arm"
                     ],
                     "supported-cycles": [
-                        "API 21+"
+                        "12",
+                        "13",
+                        "14"
                     ]
                 }
             ]
         },
         {
-            "family": "Apple",
-            "distributions": [
+            "name": "Apple",
+            "operating-system": [
                 {
+                    "id": "ios",
                     "name": "iOS",
                     "lifecycle-policy": "https://support.apple.com/ios/",
                     "architectures": [
                         "arm64"
```

Comment:
There are differences in supported architectures for iOS and the Simulator according to https://github.com/dotnet/core/blob/b488f858df2803b353233f38b16786913d6bcbc7/release-notes/9.0/supported-os.md#ios--tvos--maccatalyst which will be lost by your change.